File: app/src/main/java/com/example/indroorway/utils/NetComponent.java
package com.example.indroorway.utils;


import com.example.indroorway.baseClasses.BaseActivity;

import javax.inject.Singleton;

import dagger.Component;


@Singleton
@Component(modules = {AppModule.class, NetModule.class})
public interface NetComponent {
    void inject(BaseActivity baseActivity);
}

File: app/src/main/java/com/example/indroorway/dagger/NetComponent.kt
package com.example.indroorway.dagger


import com.example.indroorway.baseClasses.BaseActivity

import javax.inject.Singleton

import dagger.Component

@Singleton
@Component(modules = [AppModule::class, NetModule::class])
interface NetComponent {
    fun inject(baseActivity: BaseActivity)
}